### DISCUSSION TITLE
⭐️ Xcode 15.0 Release

### DISCUSSION THREAD

**Discussion by helinanever** [codemagic-ci-cd/codemagic-docs#1773] [Announcements]
**Xcode 15.0 Beta** is now available on our macOS M1 build machines. This version will be used to run the build if you specify `15`, `15.0`, or `edge` as the Xcode version.

You can configure the Xcode version in the Build section of Flutter workflow editor or in the environment section of codemagic.yaml.

See the [build machine specs](https://docs.codemagic.io/specs/versions-macos/) for information about the preinstalled software and available runtimes.

**Comment by helinanever** [comment]
We've now updated the machines with **Xcode 15.0 Beta 2**.

**Comment by helinanever** [comment]
Our build machines now have **Xcode 15.0 Beta 3** available. You can use this version to submit **visionOS apps** to TestFlight for internal and external testing, read more [here](https://developer.apple.com/news/releases/?id=07062023a).

**Comment by helinanever** [comment]
Good news - our **Xcode 15** build machines are now updated to **Beta 4**.

**Comment by mikhail-tokarev** [comment]
Late announcement but **Xcode 15.0 Beta 5** available now.

**Comment by helinanever** [comment]
**Xcode 15.0 Beta 6** is now available.

**Comment by helinanever** [comment]
Our machines now have **Xcode 15.0 Beta 7** available.

**Comment by helinanever** [comment]
We've just rolled out **Xcode 15.0 Beta 8**.

**Comment by helinanever** [comment]
We're happy to announce that our build machines now have **Xcode 15.0 RC 1** available. This update comes with a new macOS version `13.5.2` on our Xcode 15 machines and adds new runtimes, such as Apple Watch Series 9, Apple Watch Ultra 2, and iPhone 15 devices. Please note that the visionOS runtime was removed in this RC. See the full machine specs [here](https://docs.codemagic.io/specs/versions-macos/).

**Comment by helinanever** [comment]
**Xcode 15.0 Release** version is the same as RC 1, so it's already available on our machines.
